src/styles/_colours.scss

```diff
@@ -6,18 +6,18 @@
 @import "./settings";
 
 // PRIMARY COLOURS
```

Comment:
We seem to have lost the background colours for:
- evidence tags (itself but also when clicked to open)
- accordion title
- autocomplete highlight (type "item" then use up and down arrows - compare to current franklin website)
- Hero container
... basically compare with current https://ebi-uniprot.github.io/franklin-sites/ui-components

Reply:
There is actually a very slight shade of grey there, so nothing technically wrong. The colour code there is as per design.

Reply:
Would it be possible to try with one tone darker (azureish-white I think)? At the moment it's impossible to distinguish from the background. Will check with Austra aftwerwards.

Reply:
Done!

Reply:
done!